CVSym/CVSTI/DataSource.h

```diff
@@ -8,6 +8,7 @@
 #pragma once
 
 #include "IDataSource.h"
+#include <memory>

```

Comment:
What was the error here? It compiles for me in VS2017 without the include.

Reply:
It's complaining about not knowing what auto_ptr is.

Reply:
I figured out what happened.

Before VS 2017, auto_ptr was defined in xmemory, which is included in memory. But, it's also included in vector and list, which are included in Common.h. That's why the compilers didn't use to complain. Beginning in VS 2017, auto_ptr is defined in memory.

Reply:
But the question is, why does it compile for you in VS 2017?

Reply:
Probably not worth investigating further, it doesn't do much harm if added.